menas/src/test/scala/za/co/absa/enceladus/menas/integration/repositories/RunRepositoryIntegrationSuite.scala

```diff
@@ -44,6 +46,8 @@ class RunRepositoryIntegrationSuite extends BaseRepositoryTest {
     runFixture.dropCollection()
   }
 
+  private val today = LocalDate.now().format(DateTimeFormatter.ofPattern("dd-MM-yyyy"))
```

Comment:
This does not test the edge cases of what happens close to midnight

Reply:
Feel free to contribute or file an issue

Reply:
Created #541 then this can be updated. Currently, it would just fail